src/var/timeoutSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global setTimeout: false */
+	return window.setTimeout || setTimeout;
```

Comment:
But link to what exactly? I linked to my first PR, this is the third where there is no central place to put it.